Makefile

```diff
@@ -0,0 +1,25 @@
+.PHONY: build start test lint compare shell
+
+build:
+	docker build --tag ensemble-transposer .
+
+start: build
```

Comment:
Maybe this is a bad idea. If I want the files in Docker to always be up-to-date, would it be better to use a volume?